src/components/ArbeidsgiverPeriodeTabell.tsx

```diff
@@ -76,58 +82,65 @@ class ArbeidsgiverPeriodeTabell extends Component<Props, State> {
   
   getClassnameFromStatus = (status: Status): string => {
     switch (status) {
-      case Status.UNDER_BEHANDLING: return 'under-behandling';
-      case Status.AVSLÅTT: return 'avslått';
-      case Status.INNVILGET: return 'innvilget';
+      case Status.PENDING: return 'under-behandling';
+      case Status.DECLINED: return 'avslått';
+      case Status.APPROVED: return 'innvilget';
       default: return '';
     }
   };
 
   render() {
-    const { person, error, } = this.props;
+    const { i18n, t, sak, error, } = this.props;
     const { identitetsnummerSøk, sortColumn, sortDescending, fom, tom, } = this.state;
-  
-    const filteredPerioder: ArbeidsgiverPeriode[] = person?.arbeidsgiverPerioder.filter(periode => fom
-      ? periode.fom > fom!
-      : periode
-    ).filter(periode => tom
-      ? periode.tom < tom!
-      : periode
+    
+    const filteredYtelsesperioder: Ytelsesperiode[] = sak?.ytelsesperioder.filter(ytelsesperiode => fom
```

Comment:
Jeg tenker disse metodene som behandler data burde ekstraheres ut i egen klasse. Resultatet er at denne filen blir kortere og du kan teste data-algoritmene separat fra utseende. Målet til begge klassene blir tydligere.

Reply:
Yep, skulle have gjort for lenge siden, tid til cleanup